helix-term/src/lib.rs

```diff
@@ -22,3 +22,18 @@ fn true_color() -> bool {
 fn true_color() -> bool {
     true
 }
+
+pub fn set_title_from_doc(doc: &helix_view::Document) {
+    crossterm::execute!(
+        std::io::stdout(),
+        crossterm::terminal::SetTitle(format!(
+            "settitle hx {}",
```

Comment:
Are you still testing with this?

Reply:
Yes, because it works in some places, but not others